website/src/css/customTheme.css

```diff
@@ -36,4 +76,80 @@ html[data-theme='light'] .navbar-sidebar__item {
 html[data-theme='light'] .footer-copyright {
   color: #dddddd;
   font-size: smaller;
-}
\ No newline at end of file
+}
+
+
+
+/** 
+ * Used for styling introduction / front pages
+ */
+
+div.page-container {
+	max-width: var(--ifm-container-width-xl);
+	margin: auto;
+	padding: var(--ifm-spacing-horizontal);
+} 
+ 
+div.frontPageContent {
+	display: flex;
+    flex-wrap: wrap;
```

Comment:
Tabs or spaces here? It might be good for use to have a formatter to automatically correct this in another PR.